proc_blocks/fft/src/lib.rs

```diff
@@ -1,22 +1,22 @@
 #![no_std]
 
+extern crate std;
```

Comment:
Do we need this for anything?

We'll be compiling it to `wasm32-unknown-unknown` which doesn't have a standard library (bare WebAssembly doesn't even know it is running on an OS!) so you can't use `std` in this crate.

If it's just for running tests on the host machine you'll conditionally compile it in with something like this:

```rust
#[cfg(test)]
#[macro_use]
extern crate std;
```